spec/services/stsv1/patches/20250623-add-response-descriptions.patch

```diff
@@ -0,0 +1,341 @@
+diff --git a/spec/services/stsv1/oas3.patched/openapi.yaml b/spec/services/stsv1/oas3.patched/openapi.yaml
+index 5eef4d06..a3b89813 100644
+--- a/spec/services/stsv1/oas3.patched/openapi.yaml
++++ b/spec/services/stsv1/oas3.patched/openapi.yaml
+@@ -2,11 +2,20 @@ openapi: 3.1.0
+ x-id:
+   - reitit.openapi/default
+ info:
+-  title: EaaS Security Token Service
+-  summary: >-
+-    Exchange ID tokens for EaaS access tokens according to managed trust
++  title: Equinix Security Token Service
++  description: >-
++    Exchange ID tokens for Equinix access tokens according to managed trust
+     relationships.
+-  version: not specified
++  version: 1.0.0
++  termsOfService: 'https://www.equinix.com/about/legal/terms'
++  contact:
++    name: Equinix API Support
++    url: 'https://docs.equinix.com/api-support.htm'
++  license:
++    name: Equinix Inc
++    url: 'https://developer.equinix.com/agreement'
++servers:
++  - url: 'https://sts.eqix.equinix.com'
+ tags:
+   - name: use
+     description: Operations for normal users of this service
+@@ -257,20 +266,6 @@ components:
+ security:
+   - bearerAuth: []
+ paths:
+-  /.well-known/openid-configuration:
+-    get:
+-      '200':
+-        content:
+-          application/json:
+-            schema:
+-              type: object
+-  /jwks:
+-    get:
+-      '200':
+-        content:
+-          application/json:
+-            schema:
+-              type: object
+   /ops/createOidcProvider:
+     post:
+       summary: /ops/createOidcProvider
+@@ -357,6 +352,9 @@ paths:
+               additionalProperties: false
+       responses:
+         '200':
++          description: >-
++            Returns the OIDC provider that was created. The `idpId` field is the

```

Comment:
Thanks for adding the descriptions. We strive to see the descriptions and other generally applicable improvements upstreamed because the patches become brittle the more complex they are, the more areas they touch.